sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_sender_async.py

```diff
@@ -257,7 +261,7 @@ async def schedule_messages(
             )
 
     async def cancel_scheduled_messages(
-        self, sequence_numbers: Union[int, List[int]], *, timeout: Optional[float] = None
+        self, sequence_numbers: Union[int, List[int]], *, timeout: Optional[float] = None, **kwargs:Any
```

Comment:
nit & pylint failure
```suggestion
        self, sequence_numbers: Union[int, List[int]], *, timeout: Optional[float] = None, **kwargs: Any
```